github-page/_guides/quick-start-guide.md

```diff
@@ -110,15 +110,19 @@ class KittenClass {
 
 const Kitten = getModelForClass(KittenClass);
 
-let document = await Kitten.create({ name: 'Kitty' });
+let document = new Kitten({ name: 'Kitty' });
 // "document" has types of KittenClass
 ```
 
-Please note that `new Kitten({})` or `await Kitten.create({})` has no types of KittenClass, because typegoose dosnt modify functions of mongoose
+Please note that `await Kitten.create({})` has no types of KittenClass, because typegoose doesn't modify functions of mongoose
```

Comment:
why remove the new here?

Reply:
As far as I remember, new() had the types, but create doesn't.  Isn't that the case?

If not, then I guess line 114 needs the change instead I guess cause it contradicts this.

Reply:
please read the comment of https://github.com/typegoose/typegoose/pull/49#discussion_r329305000

Reply:
PS: create can be used to create multiple documents with one function call, the only type difference in that is that it returns the type as array